util/log/clog.go

```diff
@@ -37,6 +37,7 @@ import (
 	"time"
 
 	"github.com/cockroachdb/cockroach/proto"
+	"github.com/cockroachdb/cockroach/util"
```

Comment:
I'm not sure how to avoid adding this dependency other than duplicating the method and including it in both `log` and `util`. What would you suggest?